demo/DemoArticles/update.js

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env node
+/* eslint-disable require-jsdoc */
+
+const fs = require('fs')
+const request = require('request')
+const indexJSON = require('./articles.json')
+const ArticleRef = require('./ArticleRef.js').ArticleRef
+const ArticleRefSourceType = require('./ArticleRef.js').ArticleRefSourceType
+const flattenArrayOfArrays = require('../DemoUtilities.js').flattenArrayOfArrays
+
+const DATA_PATH = './data/'
+
+const articleRefs = flattenArrayOfArrays(
+  indexJSON.map(articleData => [
+    new ArticleRef(
+      articleData.lang,
+      articleData.title,
+      articleData.revision,
+      ArticleRefSourceType.mobileView
+    ),
+    new ArticleRef(
+      articleData.lang,
+      articleData.title,
+      articleData.revision,
+      ArticleRefSourceType.mobileContentService
+    )
+  ])
+)
+
+// eslint-disable-next-line no-console
+console.log(`Fetching JSON for ${articleRefs.length} titles...`)
+
+// Linting complained of some improperly formatted characters which turned out to be these.
```

Comment:
todo: try to document what's happening here better, implement this generically instead of case by case, and consider pushing this fix to the server (since they return JSON responses).